app/views/admin/attachments/_form.html.erb

```diff
@@ -1,12 +1,28 @@
 <% initialise_script "GOVUK.AdminAttachmentsForm", selector: '.js-attachment-form', right_to_left_locales: Locale.right_to_left.collect(&:to_param) %>
 
 <%= form_for attachment, url: [:admin, typecast_for_attachable_routing(attachable), attachment.becomes(Attachment)], as: :attachment, html: { :class => "js-attachment-form" } do |form| %>
-  <%= form.errors %>
-
-  <%= content_tag :fieldset, class: ("right-to-left" if form.object.rtl_locale?) do %>
-    <%= form.text_field :title %>
+  <% if form.object.errors.any? %>
+    <%= render "govuk_publishing_components/components/error_summary", {
+      title: "To save the #{form.object.class.name.demodulize.underscore.humanize.downcase} please fix the following issues:",
```

Comment:
Is this the correct variable to use?

Reply:
What does it output?

Reply:
Good question, I have no idea where that came from. It outputs the type of attachment (e.g. "external"). I've replaced it with what we've used elsewhere i.e. `#{attachment.readable_type}`